package.json

```diff
@@ -76,8 +76,8 @@
     "ml-baseline-correction-regression": "^1.0.0",
     "ml-gsd": "^6.6.1",
     "ml-savitzky-golay": "^4.1.1",
-    "ml-spectra-processing": "^5.8.0",
+    "ml-spectra-processing": "^5.9.0",
     "ml-stat": "^1.3.3",
-    "react-plot": "^0.6.0"
+    "react-plot": "^0.8.0"
```

Comment:
Since it's only used by typescript to import types, I think it could be listed in the `devDependencies`